### PR TITLE
Upgraded ARCore to 1.7.0

### PR DESCRIPTION
**Pull request by newky2k**

### CODE REVIEW

File: Android/ARCore/samples/HelloAR.sln [hunk 1]
```diff
@@ -31,4 +31,7 @@ Global
 	GlobalSection(SolutionProperties) = preSolution
 		HideSolutionNode = FALSE
 	EndGlobalSection
+	GlobalSection(MonoDevelopProperties) = preSolution
+		version = 1.7.0
+	EndGlobalSection
```
Comment (mattleibow) [inline Android/ARCore/samples/HelloAR.sln:36]: Ooh, what's this? Is this a feature that we can use at all?

File: Android/ARCore/samples/JavaGL/JavaGL.csproj [hunk 2]
```diff
@@ -8,11 +8,11 @@
     <OutputType>Library</OutputType>
     <RootNamespace>JavaGL</RootNamespace>
     <AssemblyName>JavaGL</AssemblyName>
-    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
     <MonoAndroidResourcePrefix>Resources</MonoAndroidResourcePrefix>
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
-    <AndroidUseLatestPlatformSdk>false</AndroidUseLatestPlatformSdk>
     <AndroidClassParser>class-parse</AndroidClassParser>
+    <ReleaseVersion>1.7.0</ReleaseVersion>
```
Comment (mattleibow) [inline Android/ARCore/samples/JavaGL/JavaGL.csproj]: Is this something new?

File: Android/ARCore/source/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@
 // The form "{Major}.{Minor}.*" will automatically update the build and revision,
 // and "{Major}.{Minor}.{Build}.*" will update just the revision.
 
-[assembly: AssemblyVersion("1.0.0")]
+[assembly: AssemblyVersion("1.7.0")]
```
Comment (mattleibow) [inline Android/ARCore/source/Properties/AssemblyInfo.cs]: We should probably keep this as 1.0 and then add the [AssemblyFileVersion]. This [AssemblyVersion] represents the actual assembly version and will require the app to be recompiled.

File: Android/ARCore/source/Transforms/Metadata.xml [hunk 4]
```diff
@@ -4,13 +4,15 @@
 	<attr path="/api/package[@name='com.google.ar.core.annotations']" name="managedName">Google.AR.Core.Annotations</attr>
 	<attr path="/api/package[@name='com.google.ar.core.exceptions']" name="managedName">Google.AR.Core.Exceptions</attr>
 
-	<attr path="/api/package[@name='com.google.vr.dynamite.client']" name="managedName">Google.VR.Dynamite.Client</attr>
```
Comment (mattleibow) [inline Android/ARCore/source/Transforms/Metadata.xml]: I see this is removed. Is this namespace no more? If so, then is there a reason for setting the base for `ObjectWrapper` below?

File: Android/ARCore/source/Transforms/Metadata.xml [hunk 5]
```diff
@@ -4,13 +4,15 @@
...
 	<attr path="/api/package[@name='com.google.ar.core']/class[@name='TrackableBase']" name="visibility">public</attr>
-
-	<remove-node path="/api/package[@name='com.google.ar.core']/class[@name='Point']/method[@name='getAnchors' and count(parameter)=0]" />
+    
+    <attr path="/api/package[@name='com.google.ar.core']/class[@name='AugmentedFace']/method[@name='getAnchors' and count(parameter)=0]" name="return">System.Collections.Generic.ICollection&lt;Google.AR.Core.Anchor&gt;</attr>
+	
+    <remove-node path="/api/package[@name='com.google.ar.core']/class[@name='Point']/method[@name='getAnchors' and count(parameter)=0]" />
 	<remove-node path="/api/package[@name='com.google.ar.core']/class[@name='Plane']/method[@name='getAnchors' and count(parameter)=0]" />
-	<remove-node path="/api/package[@name='com.google.ar.core']/class[@name='AugmentedImage']/method[@name='getAnchors' and count(parameter)=0]" />
+	<remove-node path="/api/package[@name='com.google.ar.core']/class[@name='AugmentedImage']/method[@name='getAnchors' and count(parameter)=0]" />\
```
Comment (mattleibow) [inline Android/ARCore/source/Transforms/Metadata.xml]:
```suggestion
	<remove-node path="/api/package[@name='com.google.ar.core']/class[@name='AugmentedImage']/method[@name='getAnchors' and count(parameter)=0]" />
```

"Dr Typo, I presume?" - Henry Morton Stanley

File: Android/ARCore/source/Transforms/Metadata.xml (hunk 4 above)
Comment (newky2k) [reply Android/ARCore/source/Transforms/Metadata.xml]: typo probably

File: Android/ARCore/source/Transforms/Metadata.xml (hunk 5 above)
Comment (newky2k) [reply Android/ARCore/source/Transforms/Metadata.xml]: fixed

**Review by mattleibow — APPROVED** [review]
LGTM

**Pull request merged**